eap74-rhel8-byos-vmss/src/test/scripts/gen-parameters.sh

```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 #read arguments from stdin
```

Comment:
Yes, we have it here. Why not in other places?

Reply:
This might be something we can ask the Brno team to apply in a comprehensive fashion.